packages/graphql/src/contracts.js

```diff
@@ -127,6 +133,14 @@ export function setNetwork(net, customConfig) {
   clearInterval(blockInterval)
 
   web3 = applyWeb3Hack(new Web3(config.provider))
+
+  if (config.useMetricsProvider) {
+    addMetricsProvider(web3, {
+      echoEvery: 100,
```

Comment:
nit: add units as a comment. Is that 100sec ? Same for breakdonwEvery.

Reply:
Figured this was covered inside the MetricsProvider source, but can't hurt to add clarifications where they're set.  Will do.